### ISSUE TITLE
Improve length of search bar on onrr.gov homepage

### ISSUE THREAD

**Issue by AlexandracmONRR** [DOI-ONRR/onrr.gov-site#2464]
Preliminary research has shown that the search input and font size should accommodate the standard user query length — typically about 27 characters. This should be tested on various devices, including different mobile devices.

Currently, the length of the search bar is approx. 20 characters.

Research this more to confirm that 27 characters is the correct length. Then, adjust search bar to the ideal number of characters for accessibility.
